lib/harvester_utils/downloader.rb

```diff
@@ -18,7 +19,10 @@ def self.url_exists?(url)
     end
 
     def self.download_file(url)
-      response = Faraday.get(url)
+      conn = Faraday.new(request: { timeout: 300 }) do |faraday|
+        faraday.adapter :httpclient
```

Comment:
Toto tu je potrebne? @tomasdrga

Reply:
ano, ten iny adapter nefailne aj po dlhsej dobe timeoutu

Reply:
Akoze asi dobre, ale pride mi, ze asi nevieme poriadne nastavit len timeout na tom povodnom adapteri.

Reply:
Ešte mi teraz napadlo ako si povedal, že možno ak už to dáva ako .zip, nebude ani treba vačší timeout. Skúsim zmeniť.